README.md

```diff
@@ -71,13 +71,26 @@ For currently supported operations, see [Commands](##Commands) section.
 
 ### From npm
 
+To install globally:
+
+```sh
+npm install --global @ethersphere/swarm-cli
+```
+
+It may require `sudo` if `npm root --global` is not writable by your
+user.
+
+Or you can install locally:
```

Comment:
I'm not sure there is a common use-case for this.

Reply:
on NixOS i can't install globally, those directories are simply not writable, not even for root.